examples/svpc_host_project/README.md

```diff
@@ -0,0 +1,34 @@
+# Shared VPC Host Project
```

Comment:
We can probably condense this into the [existing SVPC example](https://github.com/terraform-google-modules/terraform-google-project-factory/tree/master/examples/shared_vpc). I am thinking removing [this flag ](https://github.com/terraform-google-modules/terraform-google-project-factory/blob/9bc317e763f767d5666f6876fdae91b3e9a6b200/examples/shared_vpc/main.tf#L65) and ideally this addition should take care of activating the host project. WDYT?

Reply:
My initial idea is to have both examples just to avoid introducing a breaking change. Like, the ppl using the current example won't need to bother. Please, lmk your thoughts.

Reply:
@thiagonache It would not be a breaking change as it is not at the module level. We do not expect end users to directly use the example other than perhaps copy pasting.

Reply:
Got it.